rostock.py

```diff
@@ -0,0 +1,93 @@
+from bs4 import BeautifulSoup
+import urllib.request
+import re
+from datetime import datetime
+from datetime import date
+
+from pyopenmensa.feed import BaseBuilder
+from utils import Parser
+
+datePattern = re.compile(r"\w+ ([0-9]{2}\.[0-9]{2}\.[0-9]{4})")
+pricePattern = re.compile(u"(\\d+),(\\d+) €")
+
+# returns datetime.date object
+def parseDate(str):
+  match = datePattern.match(str)
+  dateStr = match.group(1)
+  dt = datetime.strptime(dateStr, "%d.%m.%Y")
+  #return dt
+  return date(dt.year, dt.month, dt.day)
+
+#def parseNotesOfMeal(td) # name = ""; for span in td.find("span"): text = firstChild; name += text; notes = span.select("small > sup"); format ist [kürzel,kürzel]
```

Comment:
It would be very nice to include the notes (ingredients and allergens). The abbreviations are all wrapped in `<strong>` and are either a number or a two-letter abbreviation.